db/migrate/20150304113723_change_column_type_for_cards.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumnTypeForCards < ActiveRecord::Migration
+  def change
+    change_column :cards, :review_date, 'date USING CAST(review_date AS date)' 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.